gis/assets/traffic_signals/README.md

```diff
@@ -1,13 +1,13 @@
-﻿# Traffic Signals dataset 
+﻿# Traffic Signals
 
-Data is updated by pulling GeoJSON files from City of Toronto Open Data.
+This is one of the [datasets managed by the Traffic Control group](https://github.com/CityofToronto/bdit_vz_programs#datasets-and-their-owners). Data is updated by pulling GeoJSON files from City of Toronto Open Data.
 
 - Red Light Cameras: https://open.toronto.ca/dataset/red-light-cameras/
 - Other traffic signals: https://open.toronto.ca/dataset/traffic-signals-tabular/
 
 Here, we set up an airflow process to automatically extract Traffic Signals data from OpenAPI and update the relevant tables in the bigdata RDS. The pipeline is in `pull_traffic_signals.py` and it pulls multiple types of traffic signals to tables `vz_safety_programs_staging.rlc`, `vz_safety_programs_staging.signals_cart`, and `gis.traffic_signal`.
```

Comment:
I think we can link to the DAG, 

https://github.com/CityofToronto/bdit_data-sources/blob/master/dags/traffic_signals.py

Reply:
DAG should be https://github.com/CityofToronto/bdit_data-sources/blob/master/dags/assets_pull.py

Reply:
link added.